### PR TITLE
fix being able to use disable commands with caps

### PR DESCRIPTION
**Pull request by DragSama**

### CODE REVIEW

File: tg_bot/modules/disable.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def disable(bot: Bot, update: Update, args: List[str]):
                 disable_cmd = disable_cmd[1:]
 
             if disable_cmd in set(DISABLE_CMDS + DISABLE_OTHER):
-                sql.disable_command(chat.id, disable_cmd)
+                sql.disable_command(chat.id, str(disable_cmd).lower())
```
Comment (PaulSonOfLars) [inline tg_bot/modules/disable.py:77]: why the `str()`? disable_cmd is already a string, no need to cast.

File: tg_bot/modules/sql/disable_sql.py [hunk 2]
```diff
@@ -57,7 +57,7 @@ def enable_command(chat_id, enable):
 
 
 def is_command_disabled(chat_id, cmd):
-    return cmd in DISABLED.get(str(chat_id), set())
+    return str(cmd).lower() in DISABLED.get(str(chat_id), set())
```
Comment (PaulSonOfLars) [inline tg_bot/modules/sql/disable_sql.py:60]: Might be worth making sure the `DISABLED` var gets initialised with lowercase commands too, to preserve backwards compatibility.

Comment (DragSama) [reply tg_bot/modules/sql/disable_sql.py:60]: So when commands are disabled, lowercase them there then add to db?

Comment (PaulSonOfLars) [reply tg_bot/modules/sql/disable_sql.py:60]: You don't need to add them to the db, just make sure that when you populate the DISABLED var, you lowercase everything.
Also, you want to remove the str() here too.

**Comment by PaulSonOfLars** [comment]
Any updates on this?

**Comment by DragSama** [comment]
> Any updates on this?

Sorry, I got busy with other project so forgot about this 😅

**Pull request closed without merging**